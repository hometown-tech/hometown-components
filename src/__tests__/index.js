import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import Button from '../components/Buttons';

test('It should return Button', () => {
  const tree = renderer.create(<Button />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('It should have these styles', () => {
  const tree = renderer.create(<Button primary />).toJSON();
  expect(tree).toHaveStyleRule('background', 'palevioletred');
});
