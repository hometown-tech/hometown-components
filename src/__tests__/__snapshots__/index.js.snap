// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should return Button 1`] = `
.c0 {
  background: white;
  color: palevioletred;
  font-size: 1em;
  margin: 1em;
  padding: 1em 2em;
  border: 2px solid palevioletred;
  border-radius: 3px;
}

@media (max-width:767px) {
  .c0 {
    background: white;
    color: red;
  }
}

<button
  className="c0"
  size="small"
/>
`;

exports[`it works 1`] = `
.c0 {
  background: white;
  color: palevioletred;
  font-size: 1em;
  margin: 1em;
  padding: 1em 2em;
  border: 2px solid palevioletred;
  border-radius: 3px;
}

@media (max-width:767px) {
  .c0 {
    background: white;
    color: red;
  }
}

<button
  className="c0"
  size="small"
/>
`;
